conda-{activate, clean, compare}: add page. conda-create: add prefix … (#18051)
tldr-pages/tldr: pages/common/conda-clean.md

# conda clean

> Delete temporary or unused files: index cache, lock files, unused cache packages, tarballs, and log files.
> More information: <https://docs.conda.io/projects/conda/en/stable/commands/clean.html>.

> - Delete [a]ll temporary or unused files [v]erbosely. Say [y]es to all confirmations:

Reviewer: ```suggestion
- Delete all temporary or unused files verbosely. Say yes to all confirmations:
```

`conda clean {{[-avy|--all --verbose --yes]}}`

> - Delete only [i]ndex cache, [t]arballs, and [l]og files:

Reviewer: ```suggestion
- Delete only index cache, tarballs, and log files:
```

`conda clean {{[-itl|--index-cache --tarballs --logfiles]}}`

- Delete only temporary [c]ache files that could not be deleted earlier due to being in-use:

> `conda clean {{[-c|--tempfiles]}} {{/path/to/tempfiles}}`

Reviewer: ```suggestion
`conda clean {{[-c|--tempfiles]}} {{path/to/tempfiles}}`
```

> - Delete only unused [p]ackages. Might delete packages installed with softlinks:

Reviewer: ```suggestion
- Delete only unused packages. Might delete packages installed with softlinks:
```

`conda clean {{[-p|--packages]}}`

> - [f]orce delete all writable packages. More broad than the `[-a|--all]` option. Will delete packages installed with softlinks:

Reviewer: ```suggestion
- Force delete all writable packages. More broad than the `[-a|--all]` option. Will delete packages installed with softlinks:
```

`conda clean {{[-f|--force-pkgs-dirs]}}`

> - Display detailed [h]elp message:

Reviewer: ```suggestion
- Display help:
```
